tests/test_rpm_verifier.py

```diff
@@ -7,24 +7,61 @@
 from pytest import MonkeyPatch
 
 from verify_rpms import rpm_verifier
-from verify_rpms.rpm_verifier import ImageProcessor, ProcessedImage, generate_output
+from verify_rpms.rpm_verifier import (
+    ImageProcessor,
+    ProcessedImage,
+    generate_output,
+    get_unsigned_rpms,
+)
+
+
+@pytest.mark.parametrize(
+    "test_input,expected",
+    [
+        pytest.param(
+            [
+                "libssh-config-0.9.6-10.el8_8 (none) RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "python39-twisted-23.10.0-1.el8ap (none) (none)",
+                "libmodulemd-2.13.0-1.el8 (none) RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
+                "gpg-pubkey-d4082792-5b32db75 (none) (none)",
+            ],
+            ["python39-twisted-23.10.0-1.el8ap"],
+            id="Mix of signed and unsigned",
+        ),
+        pytest.param(
+            [
+                "libssh-config-0.9.6-10.el8_8 (none) RSA/SHA256, Tue 6 May , Key ID 1234567890",
+                "libmodulemd-2.13.0-1.el8 (none) RSA/SHA256, Wed 18 Aug , Key ID 1234567890",
+            ],
+            [],
+            id="All signed",
+        ),
+        pytest.param([], [], id="Empty list"),
+    ],
+)
+def test_get_unsigned_rpms(test_input: list[str], expected: list[str]) -> None:
+    """Test get_unsigned_rpms"""
+    mock_runner = MagicMock()
+    mock_runner.return_value.stdout.splitlines.return_value = test_input
```

Comment:
`splitlines` is a method of the str. I think we should mock the return value of `stdout` and then test our actual usage of `splitlines`.

Reply:
Done